src/styles/stars.scss

```diff
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background: url('../images/star.svg');
+    background-repeat: no-repeat;
+    background-position: center;
+
+    width: 16px;
+    height: 16px;
+    margin-bottom: 0;
+    margin-right: 4px;
+  }
+}
+
+@for $star from 1 through 5 {
+  .stars--#{$star} {
+    .stars__star:nth-child(-n + #{$star}) {
+      background-image: url(../images/star-active.svg);
```

Comment:
The URL path for the active star background image should be checked to ensure it is correct and the image exists at the specified location.